src/pages/ProfileCreation/ProfileCreation.tsx

```diff
@@ -72,7 +72,7 @@ const ProfileCreation: React.FC = () => {
       setIsEdit(false);
       setMonogram(createMonogram(username));
       const response = await fetch(
-        process.env.BACKEND_API_URL + "username/" + username
+        process.env.BACKEND_API_URL + "/username/" + username
```

Comment:
same as above

Reply:
This is so we don't need ending slash in process.env.BACKEND_API_URL